style.css

```diff
@@ -1,3 +1,743 @@
-body{
-    background-color: red;
-}
\ No newline at end of file
+/*
+    ==============================================
+
+                ****** RESET ******
+
+    ==============================================
+*/
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
+  color: #333 !important;
+}
+
+/*
+    ==============================================
+
+                ****** GLOBAL ******
+
+    ==============================================
+*/
+
+/* grid classes */
+
+.grid-3-columns {
+  display: grid;
+  gap: 24px;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.grid-4-columns {
+  display: grid;
+  gap: 24px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(1, 1fr);
+}
+
+.hidden {
+  display: none;
+}
+
+body {
+  margin: 0 auto;
+  font-family: sans-serif;
+  color: black;
+  font-size: 14px;
+  width: 100%;
+  max-width: 1440px;
+  background-color: #f0f0f0;
+}
+
+/*
+    ==============================================
+
+                ****** HEADER ******
+
+    ==============================================
+*/
+
+header .mobile-nav {
+  display: none;
+}
+
+header nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 20px;
+}
+
+.categories-article-middle-description * {
+  color: #f42;
+}
+
+.latest-news article * {
+  margin-bottom: 10px;
+}
+
+header .header-top-nav {
+  background-color: black;
+  height: 30px;
+  font-size: 14px;
+  color: rgba(255, 255, 255, 0.6);
+}
+
+header .header-bottom-nav {
+  background-color: #fff;
+  font-size: 18px;
+  height: 48px;
+  color: #aaa;
+}
+
+header .header-top-nav .top-left-items :first-child {
+  color: white;
+}
+
+.article-middle a,
+.article-bottom a {
+  color: #696767;
+  font-weight: lighter;
+}
+
+.section-header a {
+  color: #f42;
+  font-size: 28px;
+}
+
+.categories-article-header {
+  padding: 172px 10px 10px 10px;
+}
+
+.categories-article-header p a {
+  color: #ccc;
+}
+
+.categroies-article-bottom li a {
+  color: #5a5a5a;
+  line-height: 20px;
+}
+
+header .header-top-nav .top-left-items *:not(:first-child),
+header .header-bottom-nav .bottom-left-items *:not(:first-child) {
+  margin-left: 12px;
+}
+
+.bottom-left-items li a {
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: small;
+  color: gray;
+}
+
+.latest-news p a,
+span {
+  color: #aaa;
+  font-size: 13px;
+}
+
+.bottom-left-items li a:hover {
+  color: #333;
+}
+
+header .header-top-nav .top-right-items *:not(:last-child) {
+  margin-right: 12px;
+}
+
+header .header-bottom-nav .search-icon-wrapper {
+  border-left: 1px solid #ccc;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please keep it just with the class `.top-left-items` and `.search-icon-wrappe` and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_